Code-of-Conduct-00.md

```diff
@@ -64,7 +64,7 @@ The IETF LLC will promptly investigate all reports of harassment (see separate W
  
 **7. 	Confidentiality and Proprietary Information**
  
-While much of the work of the IETF LLC is public information, Covered Individuals may access or create certain information (in written or electronic form, or communicated orally) that has been created, developed or disclosed with expectations of confidential treatment.  Examples of such confidential information include certain IETF LLC Board discussions, strategy plans, financial data, contracts, personnel data and salary information.
+The work of the IETF LLC is public by default. However, Covered Individuals may access or create certain information (in written or electronic form, or communicated orally) that has been created, developed or disclosed with expectations of confidential treatment.  Examples of such confidential information include certain IETF LLC Board discussions, strategy plans, financial data, contracts, personnel data and salary information.
```

Comment:
I would suggest just deleting the first phrase in the sentence and instead starting with "Covered Individuals may access ...." The transparency language is fully elaborated in rfc4071bis so need not be reprised here.